.gitlab-ci.yaml

```diff
@@ -0,0 +1,12 @@
+stages:
+  - integration-test
+
+polkadot-integration-test:
+  stage: integration-test
+  image: paritytech/zombienet:v1.3.91
```

Comment:
We need and image with rust to run the tests, I think we can use `paritytech/ci-unified` .